cmd/ovirt-populator/BUILD.bazel

```diff
@@ -100,21 +102,23 @@ rpmtree(
         "@glibc-common-0__2.34-82.el9.x86_64//rpm",
         "@glibc-devel-0__2.34-82.el9.x86_64//rpm",
         "@glibc-headers-0__2.34-82.el9.x86_64//rpm",
-        "@glibc-langpack-tl-0__2.34-82.el9.x86_64//rpm",
+        "@glibc-langpack-pt-0__2.34-82.el9.x86_64//rpm",
```

Comment:
bazeldnf is needlessly switching between package alternatives, same for example for libcurl below. See here for how this was solved for virt-v2v images: https://github.com/kubev2v/forklift/blob/main/hack/virt-v2v-cold-rpm-deps.sh#L28